app/src/main/java/com/example/practiceapp/UserFragment.kt

```diff
@@ -5,55 +5,26 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 
-// TODO: Rename parameter arguments, choose names that match
-// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
-private const val ARG_PARAM1 = "param1"
-private const val ARG_PARAM2 = "param2"
-
-/**
- * A simple [Fragment] subclass.
- * Use the [UserFragment.newInstance] factory method to
- * create an instance of this fragment.
- */
 class UserFragment : Fragment() {
-    // TODO: Rename and change types of parameters
-    private var param1: String? = null
-    private var param2: String? = null
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        arguments?.let {
-            param1 = it.getString(ARG_PARAM1)
-            param2 = it.getString(ARG_PARAM2)
-        }
-    }
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_user, container, false)
-    }
+        super.onCreate(savedInstanceState)
+        val v = inflater.inflate(R.layout.fragment_user, container, false)
+        val recyclerView = v.findViewById<RecyclerView>(R.id.frag_user_recyclerView)
+        val manager = LinearLayoutManager(context)
+        val decoration = DividerItemDecoration(context,manager.orientation)
+        recyclerView.layoutManager = manager
+        recyclerView.adapter = FilmAdapter(FilmsRecyclerviewProvider.films)
+        recyclerView.addItemDecoration(decoration)
```

Comment:
this code can be moved away from this state to a more proper one, check the fragment lifecycle to identify which one suits better.